### DISCUSSION TITLE
Displaying starred_at in Navidrome UI

### DISCUSSION THREAD

**Discussion by JOJ0** [navidrome/navidrome#2419] [Ideas]
It seems like I can sort by a field named starred_at via a get request:

```
http://myserver.local:4533/app/#/song?sort=starred_at&perPage=150&order=ASC&filter={%22starred%22:true}
```

I use that heavily since I regularly export my starred songs as a playlist to later import them into my DJing computer.

Just now I realized that I can't access the information saved in starred_at for a track. If I display the details of a starred track I get something like this:

![image](https://github.com/navidrome/navidrome/assets/2733783/91c36ee5-793e-4a73-9cca-a83057907a37)

Is there another way to access this information? If not I would kindly request a feature that adds this field to this details-view.

Another nice feature would be a tooltip displaying the starred_at date when hovering over the hearts/stars as we see on the very right in this screenshot / the very last column in the starred songs list.

And my final idea of how this could be implemented is to add this field to the (right-click-) selectable columns in the songs list.

**Comment by metalheim** [comment]
I agree that it would be nice to have this information available in the UI.

_Slightly unrelated_ *Regarding the export of starred songs to your DJ app:* 
It might be easier to create a smart playlist instead of manually querying the subsonic API.
A smart playlist is a text file with the ".nsp" ending and a small piece of filter code.

F.e. you can create a `starred.nsp` file with the following content:
```
{
  "all": [
    {"is": {"loved": true}}
  ],
  "sort": "dateLoved",
  "order": "asc",
  "limit": 150
}
```
Read more about that feature here: https://github.com/navidrome/navidrome/issues/1417

This `starred.nsp` file will automatically be picked up by navidrome and create a navidrome playlist called `starred` that automatically gets updated.
You can export this playlist from navidrome to an .m3u playlist file (that your dj software can read) via the WebUI (playlist -> Export) 

Or you can export to m3u via command line:
```
navidrome pls -o output.m3u -p starred
```

More details here: https://github.com/navidrome/navidrome/issues/1914#issuecomment-1362012971
